pymatgen/util/plotly_pd_layouts.py

```diff
@@ -0,0 +1,157 @@
+# coding: utf-8
```

Comment:
Looks like this file could equally be a .json file, might be a better fit? Should be easy to dump the dictionaries that are in here, and then you load them directly from the JSON in phase_diagram.py.

Reply:
Oh yes that's totally do-able and I can/will change. Is there a reason why this would be preferred though?

Reply:
Not really. More just that if it looks like a datafile, we should store it as a datafile. Could be pretty-printed JSON to make it easier to edit. Other advantages are that the JSON file doesn't have to be analyzed or linted, and reducing cross-talk between sub-modules is nice (things to do with the phase diagram plotting specifically should be stored with the phase diagram plotting). Some global Plotly templates could be put into utils though, e.g. font, etc. that might be something for us to think about.

Reply:
Got it, that all makes sense to me! Yup I think that would be a good direction to go.